scipy/optimize/_minimize.py

```diff
@@ -961,3 +972,45 @@ def standardize_constraints(constraints, x0, meth):
                 constraints.extend(old_constraints[1:])  # appends 1 if present
 
     return constraints
+
+
+def _optimize_result_for_equal_bounds(
+        fun, bounds, method, args=(), constraints=()
+):
+    """
+    Provides a default OptimizeResult for when a bounded minimization method
+    has (lb == ub).all().
+
+    Parameters
+    ----------
+    fun: callable
+    bounds: Bounds
+    method: str
+    constraints: Constraint
+    """
+    success = True
+    message = 'All independent variables were fixed by bounds.'
+
+    # bounds is new-style
+    x0 = bounds.lb
+
+    if constraints:
+        message = ("All independent variables were fixed by bounds at values"
+                   " that satisfy the constraints")
+        constraints = standardize_constraints(constraints, x0, 'new')
+
+    maxcv = 0
+    for c in constraints:
+        pc = PreparedConstraint(c, x0)
+        violation = pc.violation(x0)
+        if np.sum(violation):
+            maxcv = max(maxcv, np.max(violation))
+            success = False
+            message = (f"All decision variables were fixed by bounds, but the "
+                       f"independent variables do not satisfy the constraints "
+                       f"exactly. (Maximum violation: {maxcv}).")
```

Comment:
```suggestion
            message = (f"All independent variables were fixed by bounds, but "
                       f"the independent variables do not satisfy the "
                       f"constraints exactly. (Maximum violation: {maxcv}).")
```

Reply:
I would have fixed this, but there is another problem -some changes to PROPACK got committed here. I can't say exactly how to fix it, but Ralf's email about PROPACK and [this](https://stackoverflow.com/questions/10914022/how-do-i-check-out-a-specific-version-of-a-submodule-using-git-submodule) might be helpful. I think you need to set PROPACK to whatever commit it is set in `main`.

Reply:
Gaaah, I followed Ralf's instructions, but I still have issues it seems. I'll try and remove the PROPACK stuff.